common/spec/dependabot/pull_request_creator/message_builder/link_and_mention_sanitizer_spec.rb

```diff
@@ -281,6 +281,17 @@
       end
     end
 
+    context "with a GitHub NWO and PR number" do
+      let(:text) do
+        "dsp-testing/dependabot-ts-definitely-typed#25"
+      end
+      it do
+        is_expected.to eq(
+          "<p><code>dsp-testing/dependabot-ts-definitely-typed#25</code></p>\n"
```

Comment:
Do we need to include the newline at the end of the line?

Reply:
the [call here](https://github.com/dependabot/dependabot-core/blob/main/common/lib/dependabot/pull_request_creator/message_builder/link_and_mention_sanitizer.rb#L45) renders the doc and adds the newline, so the test check has to expect it as far as I know.